packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -967,8 +967,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
          * we don't have an op (on demand summaries for instance). In those cases, we will use the timestamp
          * of this client's connection - https://github.com/microsoft/FluidFramework/issues/8375.
          */
-        const getCurrentTimestamp = () => {
-            return this.deltaManager.lastMessage?.timestamp ?? Date.now();
+         const getCurrentTimestamp = () => {
+            const audience = this.getAudience();
```

Comment:
This variable is not needed. you can do this.getAudience().getMember(this.clientId) directly in the next line.